packages/block/src/types.ts

```diff
@@ -46,6 +46,23 @@ export interface BlockOptions {
    * Default: `false` (HF is set to whatever default HF is set by the {@link Common} instance)
    */
   setHardfork?: boolean | BigIntLike
+  /**
+   * Block parameters sorted by EIP can be found in the exported `paramsBlock` dictionary,
+   * which is internally passed to the associate `@ethereumjs/common` instance which
+   * manages parameter selection based on the hardfork and EIP settings.
+   *
+   * This option allows to provide a set of own parameters. Note that parameters
+   * get fully overwritten, so you need to blend off from the default params dict
```

Comment:
What does "blend off" mean to say here?

Reply:
Have rewritten, thought that would be somewhat of an english word. 😂